packages/cli/src/cli.ts

```diff
@@ -53,14 +53,33 @@ export async function cli(args: string[] = process.argv) {
           .demandCommand();
       },
       handler: (_args) => {},
+    })
+    .fail(async (msg, err, yargs) => {
+      const Consola = await import("consola");
+      if (err instanceof ExitProcessError) {
+        Consola.consola.error(err);
+      } else {
+        // if yargs.check returns a string it populates the err variable with a string
+        if (err && typeof err !== "string") {
+          throw err;
+        } else {
+          yargs.showHelp();
+          // eslint-disable-next-line no-console
+          console.log(""); // intentional new line
```

Comment:
Would `consola.log("")` work? Same in the other existing places that use this eslint disable